rails/7/files/.rubocop.yml

```diff
@@ -1,3 +1,81 @@
-inherit_gem:
-  gnar-style:
-    - "rubocop/rubocop_rails.yml"
+AllCops:
+  NewCops: enable
+  Exclude:
+    - db/schema.rb
+    - 'bin/**/*'
+    - 'node_modules/**/*'
+    - 'vendor/**/*'
+
+require:
+  - rubocop-performance
```

Comment:
unless we've got a project that won't use it, i'd argue that `rubocop-rspec` should also be in here.

Reply:
Adding now